mods/cnc/rules/aircraft.yaml

```diff
@@ -68,6 +68,7 @@ HELI:
 	Aircraft:
 		TurnSpeed: 7
 		Speed: 180
+		LandableTerrainTypes: Clear,Rough,Road,Ore,Beach,Tiberium,BlueTiberium
```

Comment:
Should prolly be added to the transport heli too, Chinook.

Reply:
This was copy-pasted from the transport heli; See Line 18.